README.md

```diff
@@ -24,44 +21,124 @@ You will also need to specify some configurations in your [config/config.exs]():
 ```elixir
 config :ex_plasma,
   eip_712_domain: %{
-    name: "ExPlasma",
-    salt: "some-salt",
-    verifying_contract: "contract_address",
+    name: "OMG Network",
+    salt: "0xfad5c7f626d80f9256ef01929f3beb96e058b8b4b0e3fe52d84f054c0e2a7a83",
+    verifying_contract: "0xd17e1233a03affb9092d5109179b43d6a8828607",
     version: "1"
   }
 ```
 
-## Setup (Mac OS)
+## Setup
 
-1. Clone the repo to your desktop `git@github.com:omisego/ex_plasma.git`
+1. Clone the repo to your desktop `git@github.com:omgnetwork/ex_plasma.git`
 2. Run `mix compile` in your terminal.
 3. If there are any unavailable dependencies, run `mix deps.get`.
-*If you run into any issues with* ***libsecp256k1_source***, *run* `brew install automake pkg-config libtool libffi gmp` *in your terminal.* 
-
-## Testing
-
 
-You can run the tests by running;
 
-```sh
-mix test
-mix credo
-mix dialyzer
+*If you run into any issues with* ***libsecp256k1_source***, *run in your terminal:*
+
+Mac OS: `brew install automake pkg-config libtool libffi gmp`
+Debian/Ubuntu: `apt-get -y install autoconf build-essential libgmp3-dev libtool`
+
+## Usage
```

Comment:
I think we should also show how to use the `ExPlasma.Builder` to build transactions, or you can just use it in your examples below, instead of defining the struct manually, you can build it using the `Builder`.

Reply:
I added a Builder example